### PR TITLE
Kkmattil local sacratch faq

### PR DESCRIPTION
**Pull request by kkmattil**
Local scratch FAQ tropic.

### CODE REVIEW

**Review by yetulaxman — APPROVED** [review]

File: docs/support/faq/local_scratch_for_data_processing.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+**Step 2**. Run the analysis. This time we run a for loop that uses command *transeq* to translate all the fasta files, found in the big_data directory, 
+into new protein sequence files:
+```text
+for ffile in $(find ./ | grep fasta$ )
+do
+     transeq $ffile ${ffile}.pep
+done 
+```
+In this example about 52000 fasta files where found so after the processing the big_data directory contains 52000 more small files.
```
Comment (amsaren) [inline docs/support/faq/local_scratch_for_data_processing.md:43]:
```suggestion
In this example about 52000 fasta files were found, so after the processing the big_data directory contains 52000 more small files. 
```

File: docs/support/faq/local_scratch_for_data_processing.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+```text
+for ffile in $(find ./ | grep fasta$ )
+do
+     transeq $ffile ${ffile}.pep
+done 
+```
+In this example about 52000 fasta files where found so after the processing the big_data directory contains 52000 more small files. 
+The actual translation is a simple task so relatively much time is consumed to just open and close files.
+
+**Step 3**. When the processing is finished we store the results back to scratch directory into a new tar file to the normal scratch directory
```
Comment (amsaren) [inline docs/support/faq/local_scratch_for_data_processing.md:46]:
```suggestion
**Step 3**. When the processing is finished we store the results back to scratch directory into a new tar file.
```

File: docs/support/faq/local_scratch_for_data_processing.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+```text
+tar cvf /scratch/project_2001234/big_data.pep.tar ./
+```
+Now the results are in safe in one file in scratch directory and we can exit from the interactive session.
```
Comment (amsaren) [inline docs/support/faq/local_scratch_for_data_processing.md:50]:
```suggestion
Now the results are safe in one file in scratch directory and we can exit from the interactive session.
```

File: docs/support/faq/local_scratch_for_data_processing.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+We could do the same analysis procedure in the scratch directory too.  Below is execution time comparison for running the three steps 
+above in LOCAL_SCRATCH and in normal scratch.  The response times of LOCAL_SCRATCH are be rather stable, but in the scratch directory
```
Comment (amsaren) [inline docs/support/faq/local_scratch_for_data_processing.md:53]:
```suggestion
above in LOCAL_SCRATCH and in normal scratch.  The response times of LOCAL_SCRATCH are rather stable, but in the scratch directory 
```

File: docs/support/faq/local_scratch_for_data_processing.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Generally you should avoid work flows that require creating thousands of small files. If you anyhow need to work with a huge number of files, 
+you should consider using the NVME based local temporary scratch directories, either through normal or interactive batch jobs.
+The local scratch area is visible only for the specific batch job and it is erased when the batch job ends. 
+Because of that you always first need to import your data set to the local scratch and when you finish, copy the data you want to preserve back to some more 
+permanent storage place like scratch or Allas.
+
+To demonstrate the effectivity of local scratch area let’s study a sample directory called *big_data*. The directory contains about 100 GiB of data in 120 000 files. In the beginning the data is peaked one data tar-archive file in the scratch directory of project 2001234 (/scratch/project_2001234/big_data.tar)
```
Comment (amsaren) [inline docs/support/faq/local_scratch_for_data_processing.md:21]:
```suggestion
To demonstrate the effectivity of local scratch area let’s study a sample directory called *big_data*. The directory contains about 100 GiB of data in 120 000 files. In the beginning the data is packed in one tar-archive file in the scratch directory of project 2001234 (/scratch/project_2001234/big_data.tar)
```

**Review by amsaren — CHANGES_REQUESTED** [review]

**Comment by kkmattil** [comment]
Korjattu. Kiitos,

**Review by amsaren — APPROVED** [review]

**Pull request merged**